app/Libraries/Search/BeatmapsetSearchRequestParams.php

```diff
@@ -54,10 +54,13 @@ public function __construct(array $request, ?User $user = null)
 
             $this->genre = get_int($request['g'] ?? null);
             $this->language = get_int($request['l'] ?? null);
-            $this->extra = array_intersect(
-                explode('.', $request['e'] ?? null),
-                $validExtras
-            );
+
+            $extras = explode('.', $request['e'] ?? null) ?? [];
+            $this->extra = array_intersect($extras, $validExtras);
+
+            if (in_array('bundled', $extras, true)) {
+                $this->extra[] = 'can_bundle';
+            }
```

Comment:
why does `can_bundle` even exist here?

Reply:
#5169 says it would be helpful to filter for maps that can be bundled, even if they aren't currently

Reply:
There's no filtering going on in this PR which makes adding the options kind of pointless

Reply:
🤔 filter's here, is it not working for you ?
https://github.com/ppy/osu-web/blob/da58213f23275bb0c9759fe5444b0b62b5d198d5/app/Libraries/Search/BeatmapsetSearch.php#L127-L132 ,

Reply:
I forgot about that 🤔